.travis.yml

```diff
@@ -52,24 +52,28 @@ jobs:
     include:
         # Test matrix
         - <<: *test-matrix
-          env: DMD=1.081.* F=production
+          env: DMD=1.081.* F=production CACHE_ID=$TRAVIS_BUILD_ID
```

Comment:
I guess making it global will break it, right?

Reply:
Not sure. https://docs.travis-ci.com/user/caching/#Caches-and-build-matrices implies they simply hash `env` value as a single string and use it as key - in that case globals won't matter. But doc says "all visible env vars" which would mean globals included.

Reply:
I guess it would make sense.